test/functional/feature_on_chain_government_voting_scenarios.py

```diff
@@ -161,11 +161,49 @@ def scenarios_test(self):
         self.test_scenario_66_6_percent_approval_full_no_votes(expectedStatus="Rejected")
         self.test_scenario_66_6_percent_approval_full_neutral_votes(expectedStatus="Rejected")
 
+    def scenarios_fix_test(self):
```

Comment:
Better description on the with test names please. `scenarios_fix_test` wouldn't be as helpful looking at it over a longer time horizon or as a new developer.